src/components/About/About.jsx

```diff
@@ -16,7 +16,7 @@ const About = () => {
           <p>
             <span className="spanDescription">
               <br />
-              Our Mision:
+              Our Mision:{" "}
```

Comment:
remove th extra space

Reply:
![resim](https://user-images.githubusercontent.com/44509662/130319354-4b24b214-81c9-4dd4-ba90-605fe4f346e4.png)
![resim](https://user-images.githubusercontent.com/44509662/130319361-1e07ceb6-4d5b-45a4-98fd-9d4404e9d918.png)
@halitbatur Thats why I added space, if you want I can remove it